### DISCUSSION TITLE
Firefox context menu spacing between items

### DISCUSSION THREAD

**Discussion by joefiesta** [Aris-t2/CustomCSSforFx#565] [General]
Firefox now (not sure when it started) shows very large line spacing betwen the first 3 lines of the context menu for, shall I say, general areas of the page.  (It seems different parts of the page have different context menus sometimes.)   Specifically, the context menu including "BACK   / FORWARD   / RELOAD / BOOKMARK PAGE / etc.

Do other people notice this and can this extra spacing be removed?

thanks, Joe P

**Comment by joefiesta** [comment]
edit:   Using Firefox 110.0 on Windows 7 Pro.

**Comment by Aris-t2** [comment]
Open an issue, fill the template, post screenshots.

https://github.com/Aris-t2/CustomCSSforFx/issues

![image](https://user-images.githubusercontent.com/6013569/221385978-ca58047e-b9cf-4fb6-ac8a-e021af24bd55.png)

![image](https://user-images.githubusercontent.com/6013569/221385968-4b2e5be8-65a4-4a86-b576-f0f982ff4a71.png)
